test/e2e/opm_test.go

```diff
@@ -45,6 +48,9 @@ var (
 	indexImage1 = "quay.io/olmtest/e2e-index:" + indexTag1
 	indexImage2 = "quay.io/olmtest/e2e-index:" + indexTag2
 	indexImage3 = "quay.io/olmtest/e2e-index:" + indexTag3
+
+	ocpVersion      = "v4.6"
```

Comment:
Can you explain why we hard code in OCP version here? Are we gonna update this version for every release?
Given master is 4.7 so we use 4.6 here to essentially get 4.6 and check if it is compatible with 4.7?

Reply:
oh that's because I was lazy. I need to figure out how we run it for all supported OCP versions. Not sure how not to hardcode that if you have any suggestions